pkgs/leak_tracker/README.md

```diff
@@ -1,6 +1,18 @@
 [![pub package](https://img.shields.io/pub/v/leak_tracker.svg)](https://pub.dev/packages/leak_tracker)
 [![package publisher](https://img.shields.io/pub/publisher/leak_tracker.svg)](https://pub.dev/packages/leak_tracker/publisher)
 
-Coming soon!
+Coming soon! See https://github.com/flutter/devtools/issues/3951 and https://github.com/flutter/devtools/issues/5606.
```

Comment:
I'd move this into a section below - `## Following package development` or similar.

Reply:
We do not want to recommennd to people to use the package yet, because flutter framework contains leaks. I am working on cleaning them. When it is done, I will do what you suggested.